alpha/lib/model/BatchJob.php

```diff
@@ -436,7 +436,14 @@ public function getChildJobs(Criteria $c = null)
 		BatchJobPeer::setUseCriteriaFilter(true);
 		return array_merge($result1, $result2);
 	}
-	
+
+	public function getOpenStatusChildJobs(Criteria $c = null)
+	{
+		$c ? $c = clone $c : $c = new Criteria();
```

Comment:
no need to clone the criteria.
And also better do:
$c = $c ? $c : new Criteria();